Packages/Tracking/Core/Runtime/Scripts/LeapXRServiceProvider.cs

```diff
@@ -618,8 +632,9 @@ protected virtual LeapTransform GetWarpedMatrix(long timestamp, bool updateTempo
                     break;
             }
 
+
             // Use the mainCamera parent to transfrom the warped positions so the player can move around
-            if (mainCamera.transform.parent != null)
+            if (mainCamera.transform.parent != null && _positionDeviceRelativeToMainCamera)
```

Comment:
If they have _positionDeviceRelativeToMainCamera set to false, we should fall back to using the Providers Transform.Position as the origin - rather than leaving it at the world space origin